BUILD

```diff
@@ -62,6 +62,8 @@ exports_files([
     "package.json",
     "yarn.lock",
     "VERSION",
+    "node_modules/react-date-range/dist/styles.css",
```

Comment:
I think we should just "vendor" these so they're easier to edit anyway.

You can create separate css files for them in the filter directory.